Firebase/android/app/src/main/java/com/novoda/bonfire/analytics/FirebaseAnalyticsAnalytics.java

```diff
@@ -5,35 +5,100 @@
 
 import com.google.firebase.analytics.FirebaseAnalytics;
 
-import static com.google.firebase.analytics.FirebaseAnalytics.Param.VALUE;
+import static com.google.firebase.analytics.FirebaseAnalytics.*;
 
 public class FirebaseAnalyticsAnalytics implements Analytics {
 
+    private static final String CHANNEL_NAME = "channel_name";
+    private static final String SENDER = "sender";
+    private static final String USER_ID = "user_id";
     private final FirebaseAnalytics firebaseAnalytics;
 
     public FirebaseAnalyticsAnalytics(Context context) {
-        firebaseAnalytics = FirebaseAnalytics.getInstance(context);
+        firebaseAnalytics = getInstance(context);
```

Comment:
yeah, I added a static import of `FirebaseAnalytics` because the pre-defined `Events` and `Params` are static inner classes